addons/mail/models/mail_thread.py

```diff
@@ -2796,33 +2717,30 @@ def _message_auto_subscribe(self, updated_values, followers_existing_policy='ski
         if udpated_fields:
             # fetch "parent" subscription data (aka: subtypes on project to propagate on task)
             doc_data = [(model, [updated_values[fname] for fname in fnames]) for model, fnames in updated_relation.items()]
-            res = self.env['mail.followers']._get_subscription_data(doc_data, None, None, include_pshare=True, include_active=True)
-            for fid, rid, pid, cid, subtype_ids, pshare, active in res:
+            res = self.env['mail.followers']._get_subscription_data(doc_data, None, include_pshare=True, include_active=True)
+            for _, dummy, pid, subtype_ids, pshare, active in res:
```

Comment:
`_fid`, `_rid` just to please the linter but at least still have an idea of what they contain? (note I actually don't know what `fid` and `rid` are, but it's still better than `_` and `dummy`.)

Reply:
Nope, ci/style crashes with "undefined variable". Sorry :D .

Reply:
I had the issue recently but all variables that start with `_` are ignored, even if there is something after the `_`.

Reply:
Will give it a try, thanks.